src/battle-tooltips.ts

```diff
@@ -733,6 +734,14 @@ class BattleTooltips {
 		return text;
 	}
 
+	getMoveDisplay(move: Move, pokemon: ServerPokemon) {
+		if (((move.id === 'frustration' || move.id === 'return') && move.basePower !== 102) ||
+			(move.id.startsWith('hiddenpower') && this.battle.gen < 7 && move.basePower !== 70)) {
+			return `${move.name} - ${move.basePower}BP`;
```

Comment:
```suggestion
			return `${move.name} ${move.basePower}`;
```

Reply:
You like `Frustration 98 (29/32)` vs `Frustration - 98BP (29/32)` (this was one thing I brought up on the [thread](https://www.smogon.com/forums/threads/return-frustration-base-power-tooltip.3650730/))?

Reply:
Yeah `Frustration 98 (29/32)` was always how I would have implemented it, and it matches how variable base power moves are usually talked about.

https://www.google.com/search?q="hidden+power+70"

See how people just say "Hidden Power 70" when talking about it.

Reply:
Ack. Just wanted to make sure